app/models/user.rb

```diff
@@ -13,10 +13,11 @@ class User < ApplicationRecord
                     format: { with: URI::MailTo::EMAIL_REGEXP }
   validates :password, presence: true,
                        length: { minimum: 8 },
-                       format: { with: %r{[-!$%^&*()_+|~=`{}\[\]:";'<>?,./\w]{8,}} }
+                       format: { with: %r{[-!$%^&*()_+|~=`{}\[\]:";'<>?,./\w]{8,}} },
+                       on: :create
```

Comment:
it's better to use `unless: :skip_password` and add `skip_password` `attr_reader` and use it